app/entities/landing_application_entity.rb

```diff
@@ -4,19 +4,49 @@ def initialize(landing_application)
   end
 
   def represent
+    {}.merge(application_id)
+      .merge(destination)
+      .merge(pilot)
+      .merge(permit_issed_at)
+      .merge(permit_and_registration_ids)
+      .merge(landing_and_departure_dates)
```

Comment:
To be honest I think this may be unnecessarily complicated and marginally harder to understand. But am I missing something in terms of maintainability?

Reply:
I think it's a fair comment. If it makes the code less easy to understand then it's not a good refactoring. I think this use of the  ["extract method" refactoring](https://thoughtbot.com/ruby-science/extract-method.html) is probably premature. We might want to revisit something like this once/if building the representation becomes more complex.

I've removed it for now. Thanks for pointing out that it was adding to rather than removing complexity!